doc/api/training/smd_data_parallel_release_notes/smd_data_parallel_change_log.rst

```diff
@@ -5,39 +5,64 @@ Release Notes
 #############
 
 New features, bug fixes, and improvements are regularly made to the SageMaker
-distributed data parallel library.
+data parallelism library.
```

Comment:
Qq: why did we remove `distributed` from the name here?

Reply:
The official name of the library is "the SageMaker data parallelism library. I'm trying to fix it whenever I find it.

Reply:
Reference to the AWS doc: https://docs.aws.amazon.com/sagemaker/latest/dg/data-parallel.html